pkg/agent/util/net.go

```diff
@@ -389,3 +389,23 @@ func GenerateRandomMAC() net.HardwareAddr {
 	buf[0] |= 2
 	return buf
 }
+
+func GenUplinkInterfaceName(hostIfName string) string {
+	return fmt.Sprintf("phy-%s", hostIfName)
+}
+
+func GetGlobalIPNetsByName(link *net.Interface) ([]*net.IPNet, error) {
+	addrList, err := link.Addrs()
+	if err != nil {
+		return nil, err
+	}
+	var addrs []*net.IPNet
+	for _, a := range addrList {
+		if ipNet, ok := a.(*net.IPNet); ok {
+			if ipNet.IP.IsGlobalUnicast() {
```

Comment:
Maybe we should not filter only global IP?

Reply:
I updated this part by removing this check. Instead, I modify ConfigureLinkAddresses in net_linux.go to make sure it will not return error if the address has already existed. Please help review the latest the version.